pkg/actuators/machine/utils.go

```diff
@@ -81,8 +82,8 @@ func getInstances(machine *machinev1.Machine, client awsclient.Client, instanceS
 			Values: []*string{&machineName},
 		},
 		{
-			Name:   aws.String("tag:clusterid"),
-			Values: []*string{&clusterID},
+			Name:   aws.String("tag:kubernetes.io/cluster/" + clusterID),
```

Comment:
can we move `kubernetes.io/cluster/` to a constant? can we also use use `sprintf` here?

Reply:
Sure, will make it a const.

Reply:
we could even have a function e.g `clusterKey/clusterIDTag` that returns  `fmt.Sprintf("%s%s", constant, name) see https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/aws/tags/cluster.go#L22